predicators/envs/narrow_passage.py

```diff
@@ -323,6 +324,8 @@ def _MoveAndOpenDoor_initiable(self, state: State, memory: Dict,
         # Append open door action to memory action plan
         memory["action_plan"].append(
             Action(np.array([0.0, 0.0, 1.0], dtype=np.float32)))
+        # Opening the door takes a little bit of time to plan, artificially
+        time.sleep(0.2)
```

Comment:
does this slow down any unit tests significantly? (what's the total time for unit tests before and after this PR?)